orttraining/orttraining/python/training/ortmodule/_custom_op_symbolic_registry.py

```diff
@@ -228,6 +228,8 @@ def squeeze(g, self, dim=None):
 # exporting to Split with SplitGrad as gradient graph.
 @register_symbolic("ConstantChunk", "prim")
 def prim_ConstantChunk(g, self, chunks, dim):
+    if chunks == 1:
+        return self
```

Comment:
nit: I am not sure doing this way is explicitly/clearly well supported, would using an identity (which anyway will be removed) here in case of some potential issues.  I guess this would probabbly also make is unnecessary to have chunk==1 branches in test codes.

Reply:
I was thinking this way at the beginning, this is very common when we handle ONNX graph. But maybe the exporter is different. I did search the code in PyTorch repo and they just return self instead of using Identity for the places that need no other Ops, so I followed the same way.